Core/Specifications/OfferSpecification.cs

```diff
@@ -38,16 +37,12 @@ internal class GetOffersNearRoute : Specification<Offer>
             public GetOffersNearRoute(
                 Geometry routeGeography, 
                 double dist,
-                DateTimeOffset startDate, 
-                DateTimeOffset expirationDate, 
                 PaginationFilterDTO paginationFilter)
             {
                 Query
                     .Where(offer => offer.Point.Location.IsWithinDistance(routeGeography, dist)
                     && !offer.IsClosed
-                    && offer.RelatedTripId == null
-                    && startDate <= offer.ExpirationDate
-                    && offer.StartDate <= expirationDate)
```

Comment:
we should check if offer.StartDate <= expirationDate of the trip

Reply:
Read the title of pull request, please